### PR TITLE
Debt/met 6139 techincal debt

### PR DESCRIPTION
**Pull request by stzanakis**

### CODE REVIEW

File: metis-enrichment/metis-enrichment-client/src/main/java/eu/europeana/enrichment/utils/EntityConverterUtils.java [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+
+/**
+ * Contains methods for converting enrichment entity classes to jibx entity classes
+ */
+public final class EntityConverterUtils {
+
+  private EntityConverterUtils() {
+  }
+
+  static eu.europeana.metis.schema.jibx.Organization convertOrganization(
```
[the remark this answers is not in the document]
Comment (stzanakis) [reply metis-enrichment/metis-enrichment-client/src/main/java/eu/europeana/enrichment/utils/EntityConverterUtils.java:60]: Yes you are right, usually the distinction I at least make in my head is the "Type" suffix. But it is not always there and perhaps is not clear anyway.
We can rename these methods, but I will not address this in the PR. Feel free to create a ticket to work on this on a later stage.

File: metis-repository/metis-repository-rest/src/main/java/eu/europeana/metis/repository/rest/controller/RecordController.java [hunk 2]
```diff
@@ -55,8 +55,9 @@ public class RecordController {
   public static final String CONTROLLER_TAG_NAME = "RecordController";
   private static final Logger LOGGER = LoggerFactory.getLogger(RecordController.class);
 
-  private static final Pattern UNSUPPORTED_CHARACTERS_PATTERN = Pattern.compile("[^a-zA-Z0-9_]");
+  private static final Pattern UNSUPPORTED_CHARACTERS_PATTERN = Pattern.compile("\\W");
   private static final String REPLACEMENT_CHARACTER = "_";
+  public static final String NOT_FOUND_REPORT_STRING = "No record found for this identifier.";
```
[the remark this answers is not in the document]
Comment (jeortizquan) [reply metis-repository/metis-repository-rest/src/main/java/eu/europeana/metis/repository/rest/controller/RecordController.java]: ah I see. oké.

File: metis-enrichment/metis-enrichment-client/src/main/java/eu/europeana/enrichment/utils/EntityConverterUtils.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (jeortizquan) [reply metis-enrichment/metis-enrichment-client/src/main/java/eu/europeana/enrichment/utils/EntityConverterUtils.java:60]: oké

[the remark this answers is not in the document]
Comment (jeortizquan) [reply metis-enrichment/metis-enrichment-client/src/main/java/eu/europeana/enrichment/utils/EntityConverterUtils.java:60]: https://europeana.atlassian.net/browse/MET-6228

**Pull request merged**